lms/templates/fields/field_textarea.underscore

```diff
@@ -27,7 +27,7 @@
             <% } %>
             ><%- value %></span>
         <% } else { %>
-            ><a href="#"><p class="sr"><%- screenReaderTitle %></p><span class="u-field-value-readonly" aria-hidden="false" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></span><span class="sr"><%- gettext('Click to edit') %></span></a>
+            ><button type="button"><p class="sr"><%- screenReaderTitle %></p><span class="u-field-value-readonly" aria-hidden="false" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></span><span class="sr"><%- gettext('Click to edit') %></span></button>
```

Comment:
Hm, it looks clear to me that it's actionable. Buuut, it's a bit too long. What I'll do is this:

"About me"
"Tell other learners about yourself: where you live, what your interests are, and why you're taking courses, or what you hope to learn."
<button>Add About me</button>